src/frontends/tensorflow/src/op/unary_op.cpp

```diff
@@ -43,6 +43,7 @@ template OutputVector translate_unary_op<Cosh>(const NodeContext& node);
 template OutputVector translate_unary_op<Erf>(const NodeContext& node);
 template OutputVector translate_unary_op<Exp>(const NodeContext& node);
 template OutputVector translate_unary_op<Floor>(const NodeContext& node);
+template OutputVector translate_unary_op<opset10::IsFinite>(const NodeContext& node);
```

Comment:
nit: Feels weird that some ops do not have opset specification, but some have it. I would understand if we had multiple opsets and needed older opset operation for some reason, but do we have such case? I don't think so, maybe remove `opset8`, `opset9` includes and leave only `opset10` ops without specifying opset versions for them here?

Reply:
they have opset8. It was initial approach to have the default opset. To make it explicit is out of scope of this PR.